### PR TITLE
Relax qt dependency to qt 5.7

### PR DESCRIPTION
**Pull request by martinlindhe**
First off, great work with the transition to qt!

It seems the project doesn't use any qt 5.8 specific features, and relaxing the check some would make the project more approachable.

With this change, I successfully compiled rpcs3 on Fedora 25, which is on qt 5.7.1

### CODE REVIEW

**Comment by AniLeo** [comment]
Seems like an extra commit sneaked in

**Comment by martinlindhe** [comment]
oops, rebase gone bad. PR fixed now

File: README.md [hunk 1]
```diff
@@ -46,12 +46,12 @@ To initialize the repository don't forget to execute `git submodule update --ini
 ### Configuring Qt
 
 *If you're using Visual Studio 2017 without Qt plugin support (or simply dont want to use it):* 
-1) Add `QTDIR` environment variable and set it to `<QtInstallFolder>\5.8\msvc2015_64\` </br>
+1) Add `QTDIR` environment variable and set it to `<QtInstallFolder>\5.7\msvc2015_64\` </br>
 *OR* </br>
-open the SLN, wait for projects to load, in explorer open `rpcs3qt/rpcs3qt.vcxproj.user` and set `<QTDIR>QtInstallFolder/5.8/msvc2015_64</QTDIR>`
+open the SLN, wait for projects to load, in explorer open `rpcs3qt/rpcs3qt.vcxproj.user` and set `<QTDIR>QtInstallFolder/5.7/msvc2015_64</QTDIR>`
```
Comment (Zangetsu38) [inline README.md]: this file no longer exist ^^ you can't remove this line :)

Comment (martinlindhe) [inline README.md]: as pr was merged, I sent a new pr for this change https://github.com/RPCS3/rpcs3/pull/2947

**Pull request merged**